pdns/configure-recursor

```diff
@@ -26,8 +26,17 @@ CF_PIE=""
 LD_PIE=""
 CF_FORTIFY=""
 CF_STACK=""
-BUILD_HOST="$(id -u -n)@$(hostname -f 2>/dev/null || hostname 2>/dev/null || echo localhost)"
 
+BUILD_USER="$(id -u -n)"
+case $(uname) in
+  SunOS)
+    BUILD_HOST="${BUILD_USER}@$(uname -srvp)" ;;
```

Comment:
this returns `SunOS 5.9 Generic_118558-39 sparc` which does not look like a hostname